go-controller/pkg/cni/helper_linux.go

```diff
@@ -322,7 +343,7 @@ func setupSriovInterface(netns ns.NetNS, containerID, ifName string, ifInfo *Pod
 		}
 
 		// 4. rename the host representor
-		hostIface.Name = containerID[:(15-len(ifnameSuffix))] + ifnameSuffix
```

Comment:
Could you test if SR-IOV OvS HWOL is working properly with these changes?

Reply:
Tested successfully SRIOV on primary and secondary network